tests/testApiChangeEmail.php

```diff
@@ -129,6 +138,65 @@ public function testThatSuccessfulApiCallReturnsTrue() {
 		$this->assertEmpty( self::$error_log->get() );
 	}
 
+	/**
+	 * Test that the bearer setting succeeds if there is a token stored with the correct scope.
+	 */
+	public function testThatSetBearerSucceedsWithStoredToken() {
+		$this->startHttpMocking();
+
+		set_transient( 'auth0_api_token', uniqid() );
+		set_transient( 'auth0_api_token_scope', 'update:users' );
+
+		$this->http_request_type = 'success_empty_body';
+		$api                     = new WP_Auth0_Api_Change_Email( self::$opts, self::$api_client_creds );
```

Comment:
"Test that the bearer setting succeeds". Where is this setting being called here? Maybe you are not missing the call but the name of the test method should be different and not disclose "black box" internals.

Reply:
This is basically a generic test against `set_bearer` instead of reflecting/exposing. This class requires the bearer token to be set so I'm making sure that it is and checking that it was stored. 

> Where is this setting being called here?

I'm not clear on your question.

Reply:
Not a question but rather an obversation. That the name of the test suggest you're going to call `set_bearer` but in the test you don't. It's true that you might be calling it internally when a new instance of `WP_Auth0_Api_Change_Email` is created, but the test should not know what methods are called internally. The test should only know that a new instance of `WP_Auth0_Api_Change_Email ` is being created. That's why I mentioned to only test the "black box" of the method and not its internals.